model/import/Report/WarningReportFormatter.php

```diff
@@ -34,10 +34,7 @@ class WarningReportFormatter extends AbstractReportFormatter
     public function format(array $report): Report
     {
         $reportObject = Report::createWarning(
-            __(
-                '%s line(s) are imported with warnings',
-                count($report)
-            )
+            __('%s line(s) are imported with warnings', count($report))
```

Comment:
Can we revert this change? I do not see the reason to change it.

Reply:
The reason of the change is that translation script doesn't work with previous markup. We need to fix translator in a separated tech task

cc: @boajer

Reply:
Thanks for the clarification!